cassandra/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - ahmetb
```

Comment:
definitely don't add me :) I don't know anything about Cassandra and trying to reduce my duties on this repo. :)

Reply:
done :)